src/ExpressiveInstaller/OptionalPackages.php

```diff
@@ -177,12 +177,7 @@ public static function install(Event $event)
         $installer->finalizePackage();
     }
 
-    /**
-     * @param IOInterface $io
-     * @param Composer $composer
-     * @param null|string $projectRoot
-     */
-    public function __construct(IOInterface $io, Composer $composer, $projectRoot = null)
+    public function __construct(IOInterface $io, Composer $composer, ?string $projectRoot = null)
```

Comment:
When we have default value `null` the type should be just `string`. I think we mentioned it one time on the forum...

Reply:
Just wondering... how would this work:
`public function __construct(IOInterface $io, ?string $foo = null, ?string $bar = null)`

If I want `$foo` to be `null` and `$bar` some string. As you are saying, I can't pass `null` for `$foo` as it needs a string.

e.g. `$f = new SomeClass($io, null, 'some string');`

Reply:
it will let you pass null, because when you have default `null` value the type is nullable by design